web-client/src/presenter/actions/TrialSession/formatCreateTermDatesAction.test.ts

```diff
@@ -0,0 +1,29 @@
+import { applicationContextForClient as applicationContext } from '@web-client/test/createClientTestApplicationContext';
+import { formatCreateTermDatesAction } from './formatCreateTermDatesAction';
+import { presenter } from '../../presenter-mock';
+import { runAction } from '@web-client/presenter/test.cerebral';
+
+describe('formatCreateTermDatesAction', () => {
+  beforeAll(() => {
+    presenter.providers.applicationContext = applicationContext;
+  });
+
+  it('should pass the term start date and the term end date to createISODateString', async () => {
+    await runAction(formatCreateTermDatesAction, {
+      modules: {
+        presenter,
+      },
+      state: {
+        modal: {
+          termEndDate: '03/31/2050',
+          termName: 'Test Term',
+          termStartDate: '01/01/2050',
+        },
+      },
+    });
+
+    expect(
+      applicationContext.getUtilities().createISODateString,
+    ).toHaveBeenCalledTimes(2);
```

Comment:
Is there a reason we are testing the implementation (that `createISODateString` is called) instead of the behavior (that the dates returned by `runAction` are formatted as expected)?

Reply:
You _could_ argue that the `createISODateString` logic is tested independently, and therefore doesn't need to be tested here again. However I'm not making that argument; why not just add another expectation?